### DISCUSSION TITLE
Question about "MessagePack v2 causes TypeLoadExceptions in Microsoft.AspNetCore.SignalR.StackExchangeRedis #18074"

### DISCUSSION THREAD

**Discussion by dkarakozov** [dotnet/aspnetcore#24647] [General]
Hi

I have the same problem as in described in [issue 18074](https://github.com/dotnet/aspnetcore/issues/18074). The issue was fixed in [PR 1833](https://github.com/dotnet/aspnetcore/pull/18133) and merged to master branch on 25 Jan. 
I expected this fix to be included in the [3.1.6 package](https://www.nuget.org/packages/Microsoft.AspNetCore.SignalR.StackExchangeRedis/3.1.6) which was released 24 days ago. But there were no changes. Can I expect them to be included in the next delivery for dotnet core 3.1.X or will I have to wait for .net 5?

**Comment by BrennanConroy** [comment]
That change will be in 5.0 only, we can't change versions in a patch as that would be a breaking change.

**Reply by AlexanderDemko** [reply]
.NET 5.0 is in preview only. What do you recommend to do now in prod environment where we want to use SignalR in clustered mode? Downgrade MessagePack is not the best solution.
